tests/vmlifecycle_test.go

```diff
@@ -237,43 +233,18 @@ var _ = Describe("Vmlifecycle", func() {
 	})
 })
 
-func renderPkillAllVmsJob(dockerTag string) *k8sv1.Pod {
-	job := k8sv1.Pod{
-		ObjectMeta: metav1.ObjectMeta{
-			GenerateName: "vm-killer",
-			Labels: map[string]string{
-				v1.AppLabel: "test",
-			},
-		},
-		Spec: k8sv1.PodSpec{
-			RestartPolicy: k8sv1.RestartPolicyNever,
-			Containers: []k8sv1.Container{
-				{
-					Name:  "vm-killer",
-					Image: "kubevirt/vm-killer:" + dockerTag,
-					Command: []string{
-						"pkill",
-						"-9",
-						"qemu",
-					},
-					SecurityContext: &k8sv1.SecurityContext{
-						Privileged: newBool(true),
-						RunAsUser:  new(int64),
-					},
-				},
-			},
-			HostPID: true,
-			SecurityContext: &k8sv1.PodSecurityContext{
-				RunAsUser: new(int64),
-			},
-		},
-	}
+func renderPkillAllVmsJob() *k8sv1.Pod {
 
-	return &job
+	return tests.RenderJob("vm-killer", tests.GetDockerTag(),
+		[]string{
+			"pkill",
```

Comment:
In what PID namespace is this pkill command run? Can this kill processes outside the test suite?